components/operator/internal/controllers/stack/testdata/multipod-with-audit-log-unvalid-tag/results/secrets--v1/postgres.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
If you rebase main and re-run the tests update data, they will disappear, otherwise the tests will fail